amd/src/input.js

```diff
@@ -430,11 +431,16 @@ define([
      */
     function initInputs(questionDivId, prefix, qaid, inputs) {
         var questionDiv = document.getElementById(questionDivId);
+        var language = null;
+        var langInput = document.getElementsByName(prefix + '_step_lang');
+        if (langInput.length > 0 && langInput[0].value) {
+            language = langInput[0].value;
```

Comment:
Does this work? Basically, why do I see `null` in my requests when doing preview?

Basically, started wondering after I noticed that things work in preview-popup just fine but if I change the language of Moodle in the background in another tab then the preview validation also changes language, which it should not do if this parameter is the defining factor and comes from the document inside that preview which does not change.

Reply:
Naturally, I assume that the minified scripts matched this.

Reply:
My bad. Did a last minute update to the element name to make it more unique and messed it up. Hopefully fixed now.

Reply:
Now it works as expected.